test/Concurrency/Runtime/task_deterministic_scheduling.swift

```diff
@@ -0,0 +1,38 @@
+// RUN: %target-run-simple-swift( -Xfrontend -disable-availability-checking -parse-as-library)
+
+// REQUIRES: concurrency
+// REQUIRES: executable_test
+// REQUIRES: concurrency_runtime
+
+import Darwin
+
+actor DeterministicOrderThanksToSend {
+    var counter = 0
+
+    func test_mainActor_taskOrdering() async {
+        var tasks = [(Int, Task<Void, Never>)]()
+        for iteration in 1...100 {
+            fputs("spawn:\(iteration)\n", stderr)
+
+
+            self.send {
```

Comment:
If this was `Task{}` the order of those executions is not guaranteed, and the test would fail.